Update data_tutorial.md (#40)
tradingview-pine-seeds/docs: data_tutorial.md

>__Important__
>

> > 1. Symbols are uploaded into the TradingView storage every 10 minutes.

Reviewer: Is it still accurate that the storage is updated every 10 minutes?

Author: Yes, that information is correct

> > Hence, the time of appearance of symbols on the chart is about 15 minutes.

Reviewer: ```suggestion
> Hence, symbols appear on the chart within 15 minutes.
```
